.devcontainer/scipy2023/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "jupyterlab",
+      "type": "shell",
+      "command": "jupyter lab",
+      "presentation": {
+        "reveal": "always"
+      },
+      "runOptions": {
+        "runOn": "folderOpen"
+      }
+    }
+  ]
+}
```

Comment:
This will now open up another terminal and run `jupyter lab` as soon as the codespaces open, so user would only need to `cmd + click` the link.